xml/cha_introduction.xml

```diff
@@ -318,17 +318,19 @@
   <important>
    <title>Switching plans</title>
    <para>
-    Switching from BYOS to PAYG or vice-versa is only possible on &azure;
-    through the <literal>Hybrid Benefit</literal> program. For all other cloud
-    providers, you remain on the plan you initially selected for as long as the
-    instance is running. The only way to switch plans is to start with the
-    other image and rebuild your system.
+    Switching from BYOS to PAYG or vice-versa is possible on &azure; through the
+   <literal>Hybrid Benefit</literal> program and on &gce; through <literal>Flexible
```

Comment:
s/Hybrid Benefit/Azure Hybrid Benefit/ we should use the full name of the feature